src/System.Management.Automation/engine/SpecialVariables.cs

```diff
@@ -255,6 +255,11 @@ internal static class SpecialVariables
 
         internal static readonly VariablePath InformationPreferenceVarPath = new VariablePath(InformationPreference);
 
+        internal const string NativeCommandUseErrorActionPreference = "PSNativeCommandUseErrorActionPreference";
```

Comment:
This is a lengthy variable name but I suppose it's very descriptive / intentional.

Reply:
Yeah I don't love the variable name prescribed by the RFC, but we're bound by our process there I guess.

In the [small amount of work I'd done toward this](https://github.com/rjmholt/PowerShell/commit/bab0480be7a0403c9fdce0a5307dd2cca36af854), I used `nameof` to deduplicate the variable name.

Reply:
Now using `nameof()`